test_replays/test_all.py

```diff
@@ -7,6 +7,7 @@
 
 # Newer unittest features aren't built in for python 2.6
 import sys
+
 if sys.version_info[:2] < (2, 7):
```

Comment:
Maybe not for this PR but do we really need to support Python versions < 2.7 -- They have been EOL for 5+ years.
https://devguide.python.org/devcycle/#end-of-life-branches

Reply:
I personally totally agree here.

Reply:
https://github.com/ggtracker/sc2reader#installation says 2.6 is supported!  My vote would be to drop all legacy Python support.

Reply:
Agreed. Should we open a dedicated issue for this? So others can have a word?

Reply:
done